packages/terra-functional-testing/package.json

```diff
@@ -37,6 +37,8 @@
     "@wdio/local-runner": "^6.1.4",
     "@wdio/mocha-framework": "^6.1.0",
     "@wdio/spec-reporter": "^6.0.16",
+    "@wdio/sync": "^6.1.14",
+    "axe-core": "3.5.4",
```

Comment:
There was a release this morning: https://github.com/dequelabs/axe-core/releases/tag/v3.5.5

Reply:
Impressive timing. Upgraded to latest

https://github.com/cerner/terra-toolkit/pull/416/commits/c08e020d50c63b7932fe6e0e2db42a3ec2e9efdf